src/common/snippets/include/snippets/utils/debug_caps_config.hpp

```diff
@@ -60,6 +60,18 @@ class DebugCapsConfig {
         }
     } dumpLIR;
 
+    struct : PropertyGroup {
+        std::string dir = "snippets_params_dump";
+        std::vector<std::string> snippets;
+
+        std::vector<PropertySetterPtr> getPropertySetters() override {
+            return {
+                PropertySetterPtr(new StringPropertySetter("dir", dir, "path to dumped params")),
+                PropertySetterPtr(new MultipleStringPropertySetter("snippets", snippets, "snippet names to dump params for")),
+            };
+        }
+    } dumpParams;
```

Comment:
I think we can store this field just as std::string. And please add the param parsing logic to `DebugCapsConfig::readProperties()`

Reply:
Done